lib/numbers.md

```diff
@@ -5,20 +5,24 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What are the different kinds of numbers in Ruby?
 
-Replace me with your response...
+The different kinds of numbers in Ruby are Integers and Floats.
 
 # What are some common operations and comparisons you would perform on numbers?
 
-Replace me with your response...
+Most of the common operations would be adding, subtracting, dividing and multiplying numbers. 
+There is also the option to find the remainder using modulus. 
+You can do comparisons such as the equality comparison (i.e. 4 == 5 and it would come back as false)
 
 # What is the difference between the `+` operation on a number versus on a String?
 
-Replace me with your response...
+The difference would be that if you use a '+' on a string it will add two strings together (i.e. 'apple' + 'pie' would make apple pie.
+If used in the case of numbers it will add the numbers together.
 
 # If you have a _String_ `"20"` and want to perform a mathematical operation (like division or multiplication) on it, will it work? If yes, why? If not, how would you make it work?
 
-Replace me with your response...
+It would not work properly. If done while trying to multiply it would simply write out "20" that many times (i.e. "20" * 5 = 2020202020)
+To fix it you would have to remove the quotations around the 20 and simply have 20 * 5 for example.
```

Comment:
True, removing the quotations would let you treat the `"20"` as a number instead of as a string. But how do we accomplish that in Ruby?